doc/bridges/easy-admin.rst

```diff
@@ -37,6 +37,8 @@ The integration can be configured in the ``config/packages/joli_media_easy_admin
 .. code-block:: yaml
 
     joli_media_easy_admin:
+        pagination:
+            per_page: 50  # Number of items per page
```

Comment:
can you please set it to `20`, as this is the default easyadmin pagination size?